synthesizer/process/src/finalize.rs

```diff
@@ -160,9 +164,13 @@ fn finalize_fee_transition<N: Network, P: FinalizeStorage<N>>(
     fee: &Fee<N>,
 ) -> Result<Vec<FinalizeOperation<N>>> {
     // Construct the call graph.
-    let mut call_graph = HashMap::new();
-    // Insert the fee transition.
-    call_graph.insert(*fee.transition_id(), Vec::new());
+    let call_graph = if state.block_height() >= N::CONSENSUS_V3_HEIGHT {
+        Default::default()
+    } else {
+        let mut call_graph = HashMap::new();
```

Comment:
To save some space given the new conditional wrapper, could use: `HashMap::from([(*fee.transition_id(), Vec::new())])`

Reply:
Good note!